content/docs/rfcs/65/README.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: 65
+title: 65/STATUS-ACCOUNTS
+name: Status Accounts
+status: draft
+category: Standards Track
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors:
+- Corey Petty <corey@status.im>
+- Oskar Thorén <oskar@status.im>
+- Samuel Hawksby-Robinson <samuel@status.im>
+---
+
+# Abstract
+
+This specification explains what a Status account is, and how it is created and used.
+
+# Background
+
+The core concept of an account in Status is a set of cryptographic keypairs. Namely, the combination of the following:
+1. a Waku chat identity keypair
+1. a set of cryptocurrency wallet keypairs
+
+The Status node verifies or derives everything else associated with the contact from the above items, including:
+- Ethereum address (future verification, currently the same base keypair)
+- 3 word mnemonic name
```

Comment:
We no longer make use of 3 word mnemonic names if I'm not mistaken.

Reply:
Addressed in 648fa9b, thanks!